docs-ref-conceptual/upcoming-breaking-changes-poc2.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Upcoming Breaking Changes – Azure CLI | Microsoft Docs
+description: Learn about breaking changes coming to Azure CLI in the next breaking change release
+manager: mkluck
+author: dbradish-microsoft
+ms.author: dbradish
+ms.date: 08/22/2022
+ms.topic: article
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli, seo-azure-cli
+keywords: azure cli updates, azure cli notes, azure cli versions, azure cli breaking changes
+---
+
+> [!CHANGES]
+> 1. Every breaking change message contains the following:
+>    - Timeline
+>    - Impact
+>    - Concise message
+>    - Next steps
+>    - Pointer for additional information
+> 1. Breaking changes are now grouped by command group, just like release notes
+> 1. Links to release notes have been added
+> 1. IMPORTANT!  We need to standardize on repeatable phrases like `for remediation...` and `Any script continuing to use this [something] will receive error ...`.
+
+# Upcoming breaking changes in the Azure CLI - POC2
+
+In POC2, H2s have been added for impacted command reference
+
+## account
+
```

Comment:
We might want to split the breaking change between those that will happen in the next window and those that will happen during the following ones.

Reply:
@dcaro , POC 3 (new) groups the breaking changes by date.